README.md

```diff
@@ -8,54 +8,38 @@ Android POC for www.covid-watch.org
 ## Who are we? What is this app?
 
 This repository is focused on the implementation of the Android version of the Covid Watch app. Our goals are to:
-- Allow users to anonymously record interactions with others using the [TCN Protocol](https://github.com/TCNCoalition/tcn-client-android)
+- Allow users to anonymously record interactions with others using the Google/Apple exposures notification api for Android
+    (https://github.com/google/exposure-notifications-android)
 - Notify users if someone they've interacted with in the past 2 weeks indicates they've tested positive for COVID-19 (again, anonymously!)
 - Offer a seamless UX to complete all of the above!
 
-The current version of the Figma we're working towards: https://www.figma.com/file/0uL6s79o21jwUFZz16Dr8b/Covid-Watch---App-v1.8?node-id=0%3A1
+The current version of the Figma we're working towards: (https://www.figma.com/file/I4OOjT4VhsSberDIAZTkcH/Covid-Watch---App-v2-(dev)?node-id=0%3A1
+
 
 ## Setup
 
 Clone this repo from the `develop` branch:
 
-```
 git clone git@github.com:covid19risk/covidwatch-android.git
```

Comment:
it's `covidwatch-android-en` in this repo

Reply:
Done